domain/tests/src/Unit/Access/DomainAccessCheckTest.php

```diff
@@ -0,0 +1,148 @@
+<?php
```

Comment:
Fair enough. I need to find a way to commit without it getting merged into the outstanding pull request -- which was in fact my intent. I will reorganize. and retry.
That should take care of the merge issue. I think the best thing is to reject this pull request and try again?